src/integration/java/org/humancellatlas/ingest/submission/web/ProjectStatusUpdateTest.java

```diff
@@ -49,12 +60,29 @@ public void test_statusIsInProgress_afterSubmissionCreation() throws Exception {
         Project project = createProject();
         String submissionUrl = createSubmission();
         connectSubmissionToProject(project, submissionUrl);
-        verifyProjectStatus(project, WranglingState.IN_PROGRESS);
+        verifyProjectStatus(project, IN_PROGRESS);
+    }
+
+    @Test
+    public void test_statusIsSubmitted_afterSubmissionIsExported() throws Exception {
+        Project project = createProject();
+        String submissionUrl = createSubmission();
+        connectSubmissionToProject(project, submissionUrl);
+        setSubmissionToExported(submissionUrl);
+        verifyProjectStatus(project, SUBMITTED);
+    }
```

Comment:
If can be a little verbose but I find structuring tests with **Given**, **When**, **Then** comments to be very helpful at quickly identifying what is setup for a test, what is being tested, and the test constraints. Especially with the actual test implementations are being hidden behind other methods, as has been done here.

```ProjectStatusUpdateTest.java
    @Test
     public void test_statusIsSubmitted_afterSubmissionIsExported() throws Exception {
        // Given 
         Project project = createProject();
         String submissionUrl = createSubmission();
         connectSubmissionToProject(project, submissionUrl);
         // When
         setSubmissionToExported(submissionUrl);
         // Then
         assertProjectWranglingState(project, SUBMITTED);
     }
```

Reply:
Will add to the tests.